lib/rock/webapp/tasks/api.rb

```diff
@@ -334,6 +334,26 @@ def get_operation(name_service, name, operation_name)
                         end  
                     end
                     
+                    desc "management for running tasks ('start', 'stop', 'configure', 'cleanup')"
+                    # has to be defined after other requests e.g. ':name_service/:name/properties' or ':name_service/:name/ports'
+                    # in order to be evalueated after them, otherwise this would catch the other requests and return false  
+                    get ':name_service/:name/:action' do
+                        task = task_by_name(params[:name_service], params[:name])
+                        action = params[:action]
+                        #check for allowed actions for securiry reasons
+                        #otherwise all the tasks methods could be called using this interface
+                        if ['start', 'stop', 'configure', 'cleanup'].include? action
```

Comment:
it's missing reset_exception, and it is easier on the eyes to order them by order of usage (configure, start, stop, cleanup, reset_exception)